lib/tasks/teem_tasks.rake

```diff
@@ -0,0 +1,100 @@
+require 'rake'
+
+$organization_id = 13856
+
+def remove_teem_group_member(token, tm_diff_users, all_user_ids, group_id)
+  tm_diff_users.each do |email|
+    user_id = all_user_ids[email]
+
+    user = Teem.teem_get_user(token, user_id)
+    user_groups = user["group_ids"]
+    user_groups.delete(group_id)
+
+    Teem.teem_update_group_assign(token, user_groups, user_id)
```

Comment:
Does teem_update_group_assign() update all people within one group, or all groups that one person has?